tools/build-image.sh

```diff
@@ -85,7 +85,7 @@ if ! kiwi-ng --version &>/dev/null ; then
   error_exit "missing kiwi-ng"
 fi
 
-if ! sudo mkfs.btrfs --version &>/dev/null ; then
+if ! /sbin/mkfs.btrfs --version &>/dev/null ; then
```

Comment:
Is an absolute path really necessary here?

Reply:
@votdev that's the thing, because it is in `/sbin`:
```
> mkfs.btrfs --version ; echo $?
Absolute path to 'mkfs.btrfs' is '/usr/sbin/mkfs.btrfs', so running it may require superuser privileges (eg. root).
127
```